styling-and-appearance/themes-suite/common-styling-appearance-fluent-theme.md

```diff
@@ -0,0 +1,280 @@
+
+* **CheckedBrush:** Sets the value of the background Brush applied when the element is in Checked state. It will have effect when used on elements that expose a "checked" state(ToggleButton, RadListBoxItem, etc.)  
+* **FocusBrush:** Sets the value of the background Brush applied when the element is focused.
+* **IsShadowDisabled:** The property is used to control the visibility of the shadow effect.
+* **MouseOverBrush:** Sets the value of the background Brush applied when the mouse is over the control. 
+* **PressedBrush:** Sets the value of the background Brush applied when the control is pressed.
+* **ShadowDepth:** Enum property that indicates the depth of the shadow effect over the control. There are 5 values that can be selected.
+* **CornerRadius:** Used to set the corner radius of commonly used basic controls that could need corner radius customizations but don't expose such property by default (e.g. Button, RepeatButton, ListBox, RadComboBox, etc.). **Example 11** shows how to set corner radius to a default Button element.
+
+**Example 8** shows a **RadToggleButton** control with modified brushes for its different states through the **MaterialAssist** class:
+
+#### __[XAML] Example 8: Declare the namespace for the Material components__
+{{region styling-appearance-fluent-theme-7}}
+	xmlns:mat="clr-namespace:Telerik.Windows.Controls.MaterialControls;assembly=Telerik.Windows.Controls"
+{{endregion}}
+
+#### __[XAML] Example 9: Set RadToggleButton's visual appearance through the MaterialAssist class__
+{{region styling-appearance-fluent-theme-8}}
+	  <telerik:RadToggleButton Content="RadButton" 
+                               Margin="10"
+                               mat:MaterialAssist.MouseOverBrush="{telerik:FluentResource ResourceKey=AccentMouseOverBrush}"
+                               mat:MaterialAssist.PressedBrush="{telerik:FluentResource ResourceKey=AccentPressedBrush}"
+							   mat:MaterialAssist.CheckedBrush="{telerik:FluentResource ResourceKey=ValidationBrush}"/>
+{{endregion}}
+
+#### **Figure 4: Appearance of the RadToggleButton in the different states**  
+![pressed brush](images/fluent-theme-togglebutton-pressed-brush.png)![mouseover brush](images/fluent-theme-togglebutton-mouseover-brush.png)![checked brush](images/fluent-theme-togglebutton-checked-brush.png) 
+
+## Shadow 
+
+The **Shadow** is an element that is used to add a shadow effect around another control or element. 
+
+>important The Shadow element **is not intended to directly host other elements** as the WPF drop shadow effect affects any nested texts and results in blurry characters. 
+
+If you would like to use it separately in your application, you should work with its Width/Height properties so that it appears around the desired element as shown in **Example 9**.
+
+#### **[XAML] Example 10: Using a Shadow component separately**
```

Comment:
no need to showcase the shadow here as well. It is more of a material theme thing.